src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -208,7 +208,7 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, lon
 
                 if (quicStream == null)
                 {
-                    throw new HttpRequestException(SR.net_http_request_aborted, null, RequestRetryType.RetryOnConnectionFailure);
+                    throw new HttpRequestException(HttpRequestError.Unknown, SR.net_http_request_aborted, null, RequestRetryType.RetryOnConnectionFailure);
```

Comment:
Not specific to this PR, but I'm surprised some of these are "Unknown".  Are we missing some kind of "UserRequested" / "Canceled" / "Aborted" / etc. enum status?

Reply:
I was considering whether some of these should be `ConnectionError`, but with the description of _"A transport-level failure occurred while connecting to the remote endpoint."_, it doesn't feel like the right fit.

A bunch of these are on retriable exceptions so in most cases the user won't see them, but it does feel like we're missing _something_.